python/cucim/setup.cfg

```diff
@@ -2,7 +2,7 @@
 VCS = git
 style = pep440
 versionfile_source = src/cucim/_version.py
```

Comment:
I don't think so.
`_version.py` exists in `cucim/python/cucim/src/cucim/_version.py` and when it is packaged, it would be rewritten & placed in `dist/cucim/_version.py` so only need to change `versionfile_build`

Reply:
Interesting. Ok let's leave it then :)